### ISSUE TITLE
Warnings when building the master branch html_theme, and html_static_path

### ISSUE THREAD

**Issue by Fahv** [helix-toolkit/docs#1]
When building the docs on the master branch 2 warnings are displayed

Warning: 'default' html theme has been renamed to 'classic'. Please change your html_theme setting either to the new 'alabaster' default theme, or to 'classic' to keep using the old default.

and 

WARNING: html_static_path entry u'Z:\\Projects\\docs\\static' does not exit

see below image

![masterbuildwarning](https://cloud.githubusercontent.com/assets/2633813/7782259/b0733404-00c1-11e5-8c0d-b209d6b8fa05.PNG)

**Comment by Fahv** [comment]
Solved with Pull Request 2 https://github.com/helix-toolkit/docs/pull/2

![masternowarning](https://cloud.githubusercontent.com/assets/2633813/7782303/37c1510a-00c4-11e5-9b4d-16a09aa43018.PNG)

**Comment by objorke** [comment]
I think no theme should be specified to get the nice theme at readthedocs.org
http://docs.readthedocs.org/en/latest/theme.html
I prefer the rtd theme over alabaster and classic!

**Comment by Fahv** [comment]
I agree. I have updated the pull request to not have a default theme. Having no default theme does not cause warnings.
